addons/l10n_fi_sale/__manifest__.py

```diff
@@ -5,7 +5,7 @@
     'name': 'Finland - Sale',
     'version': '1.0',
     'description': """Finland Sale""",
-    'category': 'Localization',
+    'category': 'Accounting/Localizations',
```

Comment:
See comment above, l10n_fi_sale only inherits  `sale.order` and it just requires the localization for some computation functions.

Reply:
reverted back to the Localization category